src/prefect_server/utilities/utils.py

```diff
@@ -0,0 +1,22 @@
+from typing import Iterable
+
+
+def get_chunk(iterable: Iterable, chunk_size: int):
```

Comment:
I think this would be better in a `collections.py` file. I think there may be some pushback against creating a new file for something that's only used once but 🤷

Reply:
I think I like the implementation at https://alexwlchan.net/2018/12/iterating-in-fixed-size-chunks/ more. It feels weird to build an intermediate list then cast it to a tuple. Another note that I find that name better as well as `get_chunk` makes me feel like you're returning a single chunk, I get that it's yielding but this isn't how std python names generator functions.

Reply:
We also have a dependency on `toolz` which has https://toolz.readthedocs.io/en/latest/api.html#toolz.itertoolz.partition which looks like it may be slightly less performant.

Reply:
All fair points thanks, I went with your preferred implementation in the latest commit